jina/parsers/orchestrate/runtimes/remote.py

```diff
@@ -118,6 +118,13 @@ def mixin_gateway_parser(parser):
         default='{}',
     )
 
+    parser.add_argument(
+        '--deployments-metadata',
+        type=str,
+        help='dictionary JSON with the request metadata for each Deployment',
```

Comment:
here u can say that only applies to gateway

Reply:
It is defined under the `mixin_gateway_parser`, thus it is naturally only applied to the gateway.